src/convertToObject.js

```diff
@@ -6,7 +6,15 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString.split(';').reduce((acc, prop) => {
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have same indentation to improve readability. Consider formatting the chain of `split` and `reduce` so that each method call starts on a new line with proper indentation.